CHANGES/9031.misc.rst

```diff
@@ -1 +1 @@
-Tracing overhead is avoided in the http writer when there are no active traces -- by user:`bdraco`.
```

Comment:
I think that the  passive voice in present tense is fine in many places in the change log, if that fits well the surrounding style. Giving the guidance of using the past tense is the easiest to explain but that's definitely not the only way.